client/blocks/app-promo/index.jsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import Gridicon from 'components/gridicon';
+import { sample } from 'lodash';
+
+import { recordTracksEvent } from 'state/analytics/actions';
+import store from 'store';
+import { localize } from 'i18n-calypso';
+import { noop, identity } from 'lodash';
```

Comment:
You should probably collapse these `lodash` imports into the line above where you're importing `sample`.